src/hotspot/share/runtime/os.cpp

```diff
@@ -213,8 +213,8 @@ char* os::iso8601_time(jlong milliseconds_since_19700101, char* buffer, size_t b
     abs_local_to_UTC = -(abs_local_to_UTC);
   }
   // Convert time zone offset seconds to hours and minutes.
-  const time_t zone_hours = (abs_local_to_UTC / seconds_per_hour);
-  const time_t zone_min =
+  const int zone_hours = (abs_local_to_UTC / seconds_per_hour);
+  const int zone_min =
     ((abs_local_to_UTC % seconds_per_hour) / seconds_per_minute);
```

Comment:
I think you need to cast the expression to `int` to ensure the compiler won't complain, regardless of how `time_t` may actually be declared.